.evergreen/config.yml

```diff
@@ -866,6 +922,7 @@ post:
       files:
         - "src/go.mongodb.org/mongo-driver/*.suite"
   - func: upload-mo-artifacts
+  - func: stop-load-balancer
```

Comment:
Added as a `post` task to ensure that the LB is shut down even if the tests fail.